### ISSUE TITLE
Inequalities sometimes do not shade the correct region

### ISSUE THREAD

**Issue by brettmoran** [jsxgraph/jsxgraph#84]
I noticed that when I created an inequality using the line y = x, the shaded region did not include all of the area below the line when it should have. Please see the example below:

http://jsfiddle.net/brett86/d6u4W

**Comment by kartik2112** [comment]
I dont think this issue has yet been resolved. Check out the graphs that I plotted using desmos and jsxgraph  calls:
* Desmos (An online graphing calculator) Plot:
![Desmos Plot of inequalities](https://lh3.googleusercontent.com/-rLRebAgtX1k/WS8re-cDMxI/AAAAAAAAAsA/Ys0aycgYadUTgmDQJMlxTHE783O0xGRqgCL0B/w530-d-h230-p/temp1.PNG)
* JSXGraph Plot:
![JSXGraph Plot of inequalities](https://lh3.googleusercontent.com/-bcaC_73e76A/WS8rMx7UJWI/AAAAAAAAAro/QCp1STj9uDImSr4_X5sITetRGUw7ZOIPACL0B/w316-d-h315-p/temp1.PNG)
Code used for plotting the above JSXGraph is:

            var weightMatrix = [[-2, 1, -6.5], [3,2,1], [0,-1,-1.5]];
            var lineColors = ['#3366ff', '#ff0000', '#009933'];
            var inputs = [[-2, 3, 1], [-2, 2, 1], [2, 3, 1], [2, 2, 1], [3, 4, 1], [-2, -2, 1], [2, -2, 1]];            
            var desiredOPsProps = [['^', '#3366ff'], ['^', '#3366ff'], ['o', '#ff0000'], ['o', '#ff0000'], ['o', '#ff0000'], ['[]', '#009933'], ['[]', '#009933']];
            var points = [];
            var lines = [];

            $(document).ready(function () {
                    var board = JXG.JSXGraph.initBoard('graphDiv', { axis: true, boundingbox: [-4, 5, 4, -3] });  //Creates the cartesian graph
                    var constPointSize = 5;

                    for (var i = 0; i < inputs.length; i++) {
                        points[i] = board.create('point', [inputs[i][0], inputs[i][1]], { size: constPointSize, face: desiredOPsProps[i][0], fixed: true, color: desiredOPsProps[i][1] });
                    }

                    for (var i = 0; i < weightMatrix.length; i++) {
                        lines[i] = board.create('line', [weightMatrix[i][2], weightMatrix[i][0], weightMatrix[i][1]], { strokeColor: lineColors[i], fixed: true , withLabel: true, name: function () { return weightMatrix[i][0]+'x + '+weightMatrix[i][1]+'y + '+weightMatrix[i][2]+' = 0';}, label: {position: 'lrb', offsets: [-20, 40]},display: 'internal' } );
                        ineq = board.create('inequality', [lines[i]], { fillColor: lineColors[i] });
                        
                    }
            });

**Comment by kartik2112** [comment]
Woah!! That was a really quick response! Thanks for that! 
Yes the problem has been solved now.
I cloned this version after you added the bug fix and built the jsxgraphcore.js. This is the output I am now getting:

![](https://lh3.googleusercontent.com/-pMAn-dNTFc0/WTAeczYnVUI/AAAAAAAAAuU/N80ul8Kf3jEit6SPMIyUBcxKSNP8kym6gCJoC/w309-h312-p/temp.png)

I had to add inverse:true to get it working. So I think there's some confusion left in the documentation part. 
[This documentation](https://jsxgraph.uni-bayreuth.de/docs/symbols/Inequality.html#2b703006-fd98-11e1-b79e-ef9e591c002e) states for equation of form 0 >= -3y + 2x +1 we need to mention `inverse:true` but [here (Wiki page) ](http://jsxgraph.uni-bayreuth.de/wiki/index.php/Inequalities) `inverse:true` is to be specified for -3y + 2x +1 >= 0 .

To cut down your work. According to the current implementation i.e. after this bug fix the latter one is correct.

Once again thanks a lot!
